synedrion/src/uint/traits.rs

```diff
@@ -175,7 +175,6 @@ impl HasWide for U4096 {
     type Wide = U8192;
 }
 
-// TODO(dp): Suggest crypto-bigint update nlimbs! macro.
```

Comment:
We can probably bring back `nlimbs!` usage, it doesn't produce clippy warnings anymore.